src/tpm2.c

```diff
@@ -240,14 +255,24 @@ static int TPM2_CommandProcess(TPM2_CTX* ctx, TPM2_Packet* packet,
         #endif /* !WOLFTPM2_NO_WOLFCRYPT && !NO_HMAC */
         }
 
-        /* Replace auth in session */
+        /* Place session auth */
         packet->pos = authPos;
         TPM2_Packet_AppendAuthCmd(packet, &authCmd);
         authPos = packet->pos; /* update auth position */
     }
 
-    /* Update the Auth Area size in the command packet */
-    TPM2_Packet_PlaceU32(packet, tmpSz);
+    /* Update the Auth Area total size in the command packet */
+    i = TPM2_Packet_PlaceU32(packet, authTotalSzPos);
+
```

Comment:
`(void) i;` here to prevent unused warnings when `DEBUG_WOLFTPM` is not defined

Reply:
Fixed:

```
src/tpm2.c:265:5: warning: Value stored to 'i' is never read [deadcode.DeadStores]
    i = TPM2_Packet_PlaceU32(packet, authTotalSzPos);
    ^   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 warning generated.
src/tpm2_packet.c:394:9: warning: Value stored to 'i' is never read [deadcode.DeadStores]
        i = TPM2_Packet_PlaceU32(packet, authTotalSzPos);
        ^   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```